src/socket-proxy/socket-proxyd.c

```diff
@@ -12,6 +12,7 @@
 #include "sd-daemon.h"
 #include "sd-event.h"
 #include "sd-resolve.h"
+#include "daemon-util.h"
```

Comment:
Please do not mix sd- prefixes headers and others, and sort headers.

Reply:
is it ok now?

Reply:
Yes. Thank you.